lib/elixir_security_advisories/dump.ex

```diff
@@ -1,7 +1,12 @@
 defmodule ElixirSecurityAdvisories.Dump do
   @directory "packages"
 
-  def dump(nodes) do
+  def dump({:error, message}) do
+    IO.puts(message)
```

Comment:
We are sure that the message will be inspectable?

Reply:
We could also use Logger if we want to warn the user about something

Reply:
> We are sure that the message will be inspectable?

Yes, the message will be a string.

> We could also use Logger if we want to warn the user about something

I’m changing it for `Logger.error(message)` because it’s cleaner but technically there will be no "user". The code is intended to be ran only in this repository.